pandas/conftest.py

```diff
@@ -257,7 +257,7 @@ def nselect_method(request):
 # Missing values & co.
 # ----------------------------------------------------------------
 @pytest.fixture(
-    params=[None, np.nan, pd.NaT, float("nan"), np.float("NaN"), pd.NA], ids=str
+    params=[None, np.nan, pd.NaT, float("nan"), float("nan"), pd.NA], ids=str
 )
```

Comment:
i dont think we want to `float("nan")`s in here.  maybe this should be `np.float64("nan")`?  best guess is the point of having this in addition to `np.nan` is bc `nan` is not a singleton like `pd.NaT`.

Reply:
This is equivalent to what we had before, since `np.float` was an alias for `float`. You're saying that isn't what we want?

Reply:
just speculating; im pretty confident we dont want two identical objects in here

Reply:
Ah I missed that we already had `float('"nan")`.